tests/foreman/virtwho/cli/test_nutanix.py

```diff
@@ -255,3 +258,63 @@ def test_positive_prism_central_prism_central_option(
         command = get_configure_command(virtwho_config['id'], default_org.name)
         deploy_configure_by_command(command, form_data['hypervisor-type'], org=default_org.label)
         assert get_configure_option("prism_central", config_file) == 'true'
+
+    @pytest.mark.tier2
+    def test_positive_ahv_internal_debug_option(
+        self, default_org, form_data, virtwho_config, target_sat
+    ):
+        """Verify ahv_internal_debug option by hammer virt-who-config"
+
+        :id: a169a843-6ec7-4ef1-b471-649304e77cf1268
+
+        :expectedresults:
+            1. enable-ahv-debug option has been set to no
+            2. ahv_internal_debug bas been set to false in virt-who-config-X.conf
+            3. warning message exist in log file /var/log/rhsm/rhsm.log
+            4. ahv_internal_debug option can be updated
+            5. message Host UUID {system_uuid} found for VM: {guest_uuid} exist in rhsm.log
+            6. ahv_internal_debug bas been set to true in virt-who-config-X.conf
+            7. warning message does not exist in log file /var/log/rhsm/rhsm.log
+        :CaseLevel: Integration
+
+        :CaseImportance: Medium
+
+        :BZ: 2141719
+        """
```

Comment:
I see customer case attached with this BZ
```suggestion
        :customerscenario: true
        """
```